poetry_partifact_plugin/plugin.py

```diff
@@ -0,0 +1,108 @@
+import os
+
+from cleo.events.console_command_event import ConsoleCommandEvent
+from cleo.events.console_events import COMMAND
+from cleo.events.event_dispatcher import EventDispatcher
+from cleo.io.io import IO
+from partifact.auth_token import get_token
+from partifact.config import Configuration
+from poetry.console.application import Application
+from poetry.console.commands.add import AddCommand
+from poetry.console.commands.install import InstallCommand
+from poetry.console.commands.self.self_command import SelfCommand
+from poetry.plugins.application_plugin import ApplicationPlugin
+from tomlkit import parse as parse_toml
+from tomlkit.exceptions import TOMLKitError
+
+CONFIG_PATH = "./pyproject.toml"
+
+
+class PartifactPlugin(ApplicationPlugin):  # type: ignore
```

Comment:
What's the type error here?

Reply:
This shouldn't be here (it was leftover from some code I was using as a reference for building a poetry plugin class)